test/mongorito.test.js

```diff
@@ -1120,7 +1109,45 @@ describe ('Mongorito', function () {
     secondaryDb.close();
   });
 
+  it ('be able to yield a close call (issue #23)', function * (){
+    let secondaryDb = yield Mongorito.connect('localhost/mongorito_test_2');
+    yield secondaryDb.close();
```

Comment:
Oh and I'm more than willing to add a different test here but the only outside indication of close is an event on Mongorito.instance.db.on('close') not sure how checking that helps?